src/towncrier/test/test_build.py

```diff
@@ -182,8 +182,8 @@ def test_no_newsfragment_directory(self, runner):
 
         result = runner.invoke(_main, ["--draft", "--date", "01-01-2001"])
 
-        self.assertEqual(1, result.exit_code, result.output)
-        self.assertIn("Failed to list the news fragment files.\n", result.output)
+        self.assertEqual(0, result.exit_code)
+        self.assertIn("No significant changes.\n", result.output)
```

Comment:
Thanks. It looks like this was a bug.

The test docstring states that it is expected to handle it as  no changes, but in fact it was failing :)